components/director/internal/domain/document/resolver_test.go

```diff
@@ -1 +1,201 @@
-package document
+package document_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/internal/domain/document"
+	"github.com/kyma-incubator/compass/components/director/internal/domain/document/automock"
+	"github.com/kyma-incubator/compass/components/director/internal/graphql"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestResolver_AddDocument(t *testing.T) {
+	// given
+	testErr := errors.New("Test error")
+
+	applicationID := "foo"
+	id := "bar"
+	modelDocument := fixModelDocument(applicationID, id)
+	gqlDocument := fixGQLDocument(id)
+	gqlInput := fixGQLDocumentInput(id)
+	modelInput := fixModelDocumentInput(id)
+
+	testCases := []struct {
+		Name             string
+		ServiceFn        func() *automock.DocumentService
+		ConverterFn      func() *automock.DocumentConverter
+		ExpectedDocument *graphql.Document
+		ExpectedErr      error
+	}{
+		{
+			Name: "Success",
+			ServiceFn: func() *automock.DocumentService {
+				svc := &automock.DocumentService{}
+				svc.On("Create", context.TODO(), applicationID, *modelInput).Return(id, nil).Once()
+				svc.On("Get", context.TODO(), id).Return(modelDocument, nil).Once()
+				return svc
+			},
+			ConverterFn: func() *automock.DocumentConverter {
+				conv := &automock.DocumentConverter{}
+				conv.On("InputFromGraphQL", gqlInput).Return(modelInput).Once()
+				conv.On("ToGraphQL", modelDocument).Return(gqlDocument).Once()
+				return conv
+			},
+			ExpectedDocument: gqlDocument,
+			ExpectedErr:      nil,
+		},
+		{
+			Name: "Create Error",
```

Comment:
here you can use sentences, so instead of `Create Error`, you can use `Returns error when document creation failed`

Reply:
I changed error test names to more descriptive ones.